docs/01-overview/02-main-areas/service-management/smgt-14-hb-create-addons-repo.md

```diff
@@ -0,0 +1,295 @@
+```
+
+  </details>
+  <details>
+  <summary>
+  Mercurial
+  </summary>
+
+If you want to use Mercurial (hg), place your addons directly in addons directories. The repository structure looks as follows:
+```
+sample-addon-repository
+  ├── {addon_x_name}-{addon_x_version}               # An addon directory
+  ├── {addon_y_name}-{addon_y_version}        
+  ├── ...                                      
+  ├── index.yaml                                     # A file which defines available addons
+  ├── index-2.yaml                              
+  └── ...                                                    
+```
+> **NOTE:** The amount of memory and storage size determine the maximum size of your addons repository. These limits are set in the
+[Helm Broker chart](../../../05-technical-reference/05-configuration-parameters/smgt-03-hb-chart.md).
+
+You can specify a Mercurial repository URL by adding a special `hg::` prefix to the URL address. After this prefix, provide a valid Mercurial URL with one of the supported protocols. In the URL, you can specify a revision to checkout.
+
+These are the allowed addon repository URLs provided in CAC or AC custom resources for Mercurial:
+```yaml
+apiVersion: addons.kyma-project.io/v1alpha1
+kind: ClusterAddonsConfiguration
+metadata:
+  name: addons-cfg-sample
+spec:
+  repositories:
+    # Mercurial HTTPS protocol with a path to index.yaml
+    - url: "hg::https://hg.osdn.net/view/project-name/repo-name//index.yaml"
+    # Mercurial HTTPS protocol with a path to index.yaml and a revision
+    - url: "hg::https://hg.osdn.net/view/project-name/repo-name//index.yaml?rev=e67e535230d4eded318b30967e32397872e53af1"
+```
+
+  </details>
+
+</div>
+
+## Supported protocols with authentication
+
+You can provide authentication as part of the URL in your AddonsConfiguration and ClusterAddonsConfiguration custom resources. Instead of using sensitive information directly in the URL, put it in the Secret resource and take advantage of templating. In your repository URL, use placeholders which refer to keys in the Secret. See the following example:
+
+```yaml
+apiVersion: addons.kyma-project.io/v1alpha1
+kind: ClusterAddonsConfiguration
+metadata:
+  name: addons-cfg-sample
+spec:
+  repositories:
+    - url: "https://{host}/{project}/addons/index.yaml"
+      secretRef:
+        name: data
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: data
+type: Opaque
+stringData:
+  host: "github.com"
+  project: "kyma-project/addons"       
+```
+The URL renders as follows:
+```
+https://github.com/kyma-project/addons/addons/index.yaml
+```
+
+The Helm Broker supports authentication with these protocols:
+
+<div tabs>
+  <details>
+  <summary>
+  HTTP/HTTPS
+  </summary>
+
+To secure your addons repository with basic authentication credentials, create a Secret resource which contains credentials, and reference it in the repository URL definition using templating. Follow these steps:
+
```

Comment:
now this is definitely a sequence of instructions :D Why not put it under "tutorials" ?